pkg/destroy/aws/aws.go

```diff
@@ -520,14 +525,19 @@ func deleteEC2InternetGateway(client *ec2.EC2, id string, logger logrus.FieldLog
 
 	for _, gateway := range response.InternetGateways {
 		for _, vpc := range gateway.Attachments {
+			if vpc.VpcId == nil {
+				logger.WithField("gateway", id).Warn("gateway does not have a vpc id")
+				continue
+			}
+			vpcID := *vpc.VpcId
```

Comment:
You've just checked to ensure `*vpc.VpcId` is non-`nil`.  Why store it in an environment variable?